ios/IJKMediaPlayer/IJKMediaPlayer/ijkmedia/ijksdl/ios/IJKSDLAudioQueueController.m

```diff
@@ -41,6 +41,28 @@ @implementation IJKSDLAudioQueueController {
     NSLock *_lock;
 }
 
+- (void) allocateAudioQueueBuffers {
+    for (int i = 0; i < kIJKAudioQueueNumberBuffers; i++)
+    {
+        if (!_audioQueueBufferRefArray[i]) {
+            AudioQueueAllocateBuffer(_audioQueueRef, _spec.size, &_audioQueueBufferRefArray[i]);
+            _audioQueueBufferRefArray[i]->mAudioDataByteSize = _spec.size;
+            memset(_audioQueueBufferRefArray[i]->mAudioData, 0, _spec.size);
+            AudioQueueEnqueueBuffer(_audioQueueRef, _audioQueueBufferRefArray[i], 0, NULL);
+        }
+    }
+}
+
+- (void) freeAudioQueueBuffers {
```

Comment:
freeAudioQueueBuffers is used just in my code logic, it is really no need after AudioQueueDispose, but I think it is needed after AudioQueueReset